go.mod

```diff
@@ -1,91 +1,97 @@
 module github.com/conductorone/baton-confluence
 
-go 1.20
+go 1.21
 
 require (
-	github.com/conductorone/baton-sdk v0.1.7
+	github.com/conductorone/baton-sdk v0.2.6
```

Comment:
This should probably be the latest release of baton-sdk since we fixed environment variables in 0.2.7 or so.